config/config_helpers_test.go

```diff
@@ -316,4 +317,86 @@ func TestResolveTerragruntConfigString(t *testing.T) {
 			assert.Equal(t, testCase.expectedOut, actualOut, "For string '%s' include %v and options %v", testCase.str, testCase.include, testCase.terragruntOptions)
 		}
 	}
+}
+
+func TestResolveEnvInterpolationConfigString(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		str			   string
+		include		   *IncludeConfig
+		terragruntOptions options.TerragruntOptions
+		expectedOut	   string
+		expectedErr	   error
+		env			   map[string]string
+	}{
+		{
+			"foo/${get_env()}/bar",
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/.terragrunt", NonInteractive: true},
+			"",
+			InvalidFunctionParameters(""),
+			nil,
+		},
+		{
+			"foo/${get_env(Invalid Parameters)}/bar",
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/.terragrunt", NonInteractive: true},
+			"",
+			InvalidFunctionParameters("Invalid Parameters"),
+			nil,
+		},
+		{
+			"foo/${get_env(ENV,\"\")}/bar",
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/.terragrunt", NonInteractive: true},
+			"",
+			InvalidFunctionParameters("ENV,\"\""),
+			nil,
+		},
+		{
+			"foo/${get_env(TEST_ENV_TERRAGRUNT_HIT,'')}/bar",
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/.terragrunt", NonInteractive: true},
+			"foo//bar",
+			nil,
+			map[string]string{"TEST_ENV_TERRAGRUNT_OTHER": "SOMETHING"},
+		},
+		{
+			"foo/${get_env(TEST_ENV_TERRAGRUNT_HIT,'DEFAULT')}/bar",
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/.terragrunt", NonInteractive: true},
+			"foo/DEFAULT/bar",
+			nil,
+			map[string]string{"TEST_ENV_TERRAGRUNT_OTHER": "SOMETHING"},
+		},
+		{
+			"foo/${get_env(TEST_ENV_TERRAGRUNT_HIT,'default')}/bar",
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/.terragrunt", NonInteractive: true},
+			"foo/HIT/bar",
+			nil,
+			map[string]string{"TEST_ENV_TERRAGRUNT_HIT": "HIT"},
```

Comment:
Do any of these test cases validate that `get_env` falls back to the default value?